packages/sf-core/prepareReleaseTars.sh

```
@@ -23,10 +23,10 @@ node prepareDistributionTarballs.js
 cd ../../framework-dist
 npm pack
 if [ "$is_canary" = true ]; then
-    aws s3 cp ./serverlessinc-framework-alpha-${version}.tgz s3://${s3_bucket}/archives/canary-${version}.tgz
-    aws s3 cp ./serverlessinc-framework-alpha-${version}.tgz s3://${s3_bucket}/archives/canary.tgz
+    aws s3 cp ./serverless-npm-${version}.tgz s3://${s3_bucket}/archives/canary-${version}.tgz
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)